src/hotspot/share/nmt/memTracker.inline.hpp

```diff
@@ -23,11 +23,11 @@
  *
  */
 
-#ifndef SHARE_SERVICES_MEMTRACKER_INLINE_HPP
-#define SHARE_SERVICES_MEMTRACKER_INLINE_HPP
+#ifndef SHARE_NMT_MEMTRACKER_INLINE_HPP
+#define SHARE_NMT_MEMTRACKER_INLINE_HPP
 
-#include "services/mallocTracker.inline.hpp"
-#include "services/memTracker.hpp"
+#include "nmt/mallocTracker.inline.hpp"
+#include "nmt/memTracker.hpp"
```

Comment:
Bonus points if you fix the include for this .inline.hpp file.
```suggestion
#include "nmt/memTracker.hpp"

#include "nmt/mallocTracker.inline.hpp"
```